projects/vdk-heartbeat/src/vdk/internal/heartbeat/vdk_heartbeat_data_job/trino_ingestion/10_ingest_data.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def run(job_input: IJobInput):
+    props = job_input.get_all_properties()
+
+    payload = {"id1": props["id1"], "id2": props["id2"], "id3": props["id3"]}
+
+    # Ingest the data
+    log.info(f"Sending the following payload for ingestion: {payload}")
+    job_input.send_object_for_ingestion(
+        payload=payload, destination_table=props["destination_table"], method="http"
```

Comment:
vdk-trino provides its own ingestion method: https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/plugins/vdk-trino/src/vdk/plugin/trino/ingest_to_trino.py 

vdk-ingest-http provides a method for ingesting via http - https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-core/plugins/vdk-ingest-http 

Those are different. 

It is possible to ingest via http into trino database indirectly as we do in our internal deployment or via segment or other similar service (you can send to segment via HTTP -> which can write to trino for example). 

But that's separate case.

Reply:
So to summarize. Our testing job needs to be able to ingest data into a database and read from that database (to verify the results). 

So ingest_method_default and db_default_type need to be set to ultimately go to the same database. 

This can happen in following cases 

A) ingest_method_default is set to "TRINO"  and db_default_type is set to "TRINO" and the connection string for both is the same (same trino database instance)
B) ingest_method_default is set to "HTTP" or "segment" (which writes to trino)  and db_default_type and connection settings are set to same trino instance 

Trino is used as an example. it could be any data warehouse - redshift, impala, snowflake.

Reply:
Could you please check whether it already covers this requirment.

Reply:
Can you link the issue? What problem are we trying to solve?